scipy/io/tests/test_netcdf.py

```diff
@@ -45,6 +45,22 @@ def check_simple(ncfileobj):
     assert_equal(time.shape, (N_EG_ELS,))
     assert_equal(time[-1], N_EG_ELS-1)
 
+def assert_mask_matches(arr, expected_mask):
+    '''
+    Asserts that the mask of arr1 is effectively the same as expected_mask.
+
+    In contrast to numpy.ma.testutils.assert_mask_equal, this function allows
+    testing the 'mask' of a standard numpy array (the mask in this case is treated
+    as all False).
+
+    Arguments:
```

Comment:
This is normally formatted like so:

    Parameters
    ----------
    arr: ndarray or MaskedArray
        Array to test.
    expected_mask: list of bools
        A list giving the expected mask.